libs/@local/hashql/core/src/module/std_lib/core/graph/body.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{
+    heap::Heap,
+    module::{
+        StandardLibrary,
+        locals::TypeDef,
+        std_lib::{self, ModuleDef, StandardLibraryModule, core::func, decl},
+    },
+    symbol::Symbol,
+};
+
+pub(in crate::module::std_lib) struct Body {
+    _dependencies: (std_lib::core::graph::Graph,),
+}
+
+impl<'heap> StandardLibraryModule<'heap> for Body {
+    type Children = ();
+
+    fn name(heap: &'heap Heap) -> Symbol<'heap> {
+        heap.intern_symbol("body")
+    }
+
+    fn define(lib: &mut StandardLibrary<'_, 'heap>) -> ModuleDef<'heap> {
+        let mut def = ModuleDef::new();
+        let heap = lib.heap;
+
+        let graph = lib.manifest::<std_lib::core::graph::Graph>();
+
+        let mut graph_param = graph.expect_type(heap.intern_symbol("Graph"));
+        let mut graph_returns = graph_param;
+
+        graph_param.instantiate(&mut lib.instantiate);
+        graph_returns.instantiate(&mut lib.instantiate);
+
+        // `filter<T>(graph: Graph<T>, predicate: fn(entity: T) -> bool) -> Graph<T>;`
```

Comment:
To prevent confusing with `Entity` we should use
```suggestion
        // `filter<T>(graph: Graph<T>, predicate: fn(element: T) -> bool) -> Graph<T>;`
```
or
```suggestion
        // `filter<T>(graph: Graph<T>, predicate: fn(vertex: T) -> bool) -> Graph<T>;`
```